src/cbc_sdk/platform/__init__.py

```diff
@@ -8,7 +8,9 @@
 
 from cbc_sdk.platform.alerts import Alert as BaseAlert
 
-from cbc_sdk.platform.audit import AuditLog
+from cbc_sdk.platform.audit import AuditLogRecord
+
+from cbc_sdk.platform.audit import AuditLogRecord as AuditLog
```

Comment:
Why do we need to rename this?

Reply:
Because the name of the class is misleading; it was `AuditLog` but it really represents a single _record_ in the audit log, hence, `AuditLogRecord`. The old name must be kept around, however, for backward compatibility.

Reply:
AuditLog is both plural and singular. It is an Audit Log

Reply:
So, you're saying that the class _shouldn't_ be renamed?

Reply:
Yes I am suggesting we don't rename the class unless we have a functional reason why we need a different name